signer/src/error.rs

```diff
@@ -444,6 +454,15 @@ pub enum Error {
     #[error("invalid configuration")]
     InvalidConfiguration,
 
+    /// We throw this when signer produced txid, and txid coordinator sent are different
```

Comment:
I would rephrase this a little:
```suggestion
    /// We throw this when signer produced txid and coordinator produced txid differ.
```